lua/amazonq/util.lua

```diff
@@ -89,6 +93,39 @@ function M.dismiss_popup()
   popup_win = nil
 end
 
+--- Injects SSO authentication behavior into an existing popup window
+--- @param message string The popup message.
+--- @return boolean true if SSO behavior was injected
+function M.on_sso_msg(message)
```

Comment:
Not a blocker: I know I suggested `on_` before, but now that I understand this better it seems clear that it's just attempting to format stuff, so `try_format_...` is a good name.


```suggestion
function M.try_format_sso_msg(message)
```

Reply:
Yea ... I was resisting changing it to avoid churn, but I like your name suggestion.

Changed in the latest version.